emoji-ui/src/components/SourceList.js

```diff
@@ -26,9 +27,9 @@ class SourceListBase extends Component {
   }
 
   render() {
-    const { sourceEmoji } = this.props;
+    const { sourceEmoji, sourceEmojiActions } = this.props;
     const firstBatch = sourceEmoji.slice(0, 200);
-    return <EmojiList sourceEmoji={firstBatch} />;
+    return <EmojiList sourceEmojiActions={sourceEmojiActions} sourceEmoji={firstBatch} />;
```

Comment:
this is why i think sourceList and emojiList should merge -- just do pagination in selector and boom! no more need for this wrapping!

Reply:
sounds good! will be working toward removing the SourceList wrapper.